pyproject.toml

```diff
@@ -65,11 +65,11 @@ Werkzeug = "3.0.4"
 MarkupSafe = "2.1.5"
 # REVIEW: v2 is using sha512 instead of sha1 by default (in v1)
 itsdangerous = "2.2.0"
-notifications-utils = { git = "https://github.com/cds-snc/notifier-utils.git", branch = "task/pre-3.12-recipient-csv-annual-limit-validation"}
```

Comment:
once we've verified this code is working we'll merge in the utils PR and switch api to use a proper release tag of utils